components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/session/extender/request/SessionExtenderRequestFactoryTest.java

```diff
@@ -106,7 +103,7 @@ public void testCanHandle(String requestURI, boolean canHandleResult) {
         return new Object[][]{
                 {IDP_SESSION_KEY, null, IDP_SESSION_KEY, null},
                 {null, new Cookie[] {sessionCookie}, null, sessionCookie},
-                {IDP_SESSION_KEY, new Cookie[] {sessionCookie}, IDP_SESSION_KEY, null}
+                {IDP_SESSION_KEY, new Cookie[] {sessionCookie}, IDP_SESSION_KEY, sessionCookie}
```

Comment:
Why we change null to sessionCookie?

Reply:
If needed we can have additional data entry. no need to manipulate existing one.

Reply:
There is an issue with the test case (dataProvider). 
When the sessionExtenderRequestFactory.create method called with the parameter mockedHttpRequest where, 
        (mockedHttpRequest.getCookies()).thenReturn(requestCookies);
The builded resultant object gives the corresponding cookie (here sessionCookie, not null) when getSessionCookie() method calls.